Algorithm.Framework/Selection/ManualUniverse.cs

```diff
@@ -28,6 +29,9 @@ namespace QuantConnect.Algorithm.Framework.Selection
     /// </summary>
     public class ManualUniverse : UserDefinedUniverse
     {
+        /// <summary>
+        /// Creates a new instance of the ManualUniverse
```

Comment:
not need to change, in future please use the xml reference -- this provides linking behavior.
```
<see cref="ManualUniverse"/>
```

Reply:
I'll add it, also I'll add the obsolete and new constructor 👍